db.py

```diff
@@ -13,3 +15,23 @@
 )
 
 meta.create_all(engine)
+
+
+def measurements_insert(**kwargs):
+    conn = engine.connect()
+    ins = measurements.insert().values(**kwargs)
+    conn.execute(ins)
+
+
+def last_n_measurements(n):
```

Comment:
l think we should use preset MONITORED_VALUES to prevent code duplication.

Reply:
you're right, will be done